packages/evolution-frontend/src/components/inputs/__tests__/__snapshots__/InputMapFindPlace.test.tsx.snap

```diff
@@ -27,6 +27,11 @@ exports[`Render InputMapPoint with various parameters Test with all parameters 1
       onKeyDown={[Function]}
       onMouseDown={[Function]}
       onMouseUp={[Function]}
+      style={
+        {
+          "visibility": "hidden",
```

Comment:
C'est pas plutôt display: none ? 

https://www.freecodecamp.org/news/css-display-none-and-visibility-hidden-the-difference/

Reply:
aha! je pense que le display: none l'obligerait à mocker InputMapGoogle pour forcer le rendre... ;-)

Reply:
Non, ca a l'air de toujours marcher. Je vais tester encore un peu mais si tous marche je pense que je vais fare ca a la place parce que en terme de UI ca fait plus de sens qu'il y ait pas d'espace vide tans que le bouton n'apparait pas